core/app/models/spree/store.rb

```diff
@@ -58,6 +58,10 @@ def self.available_locales
       end
     end
 
+    def default_menu(location)
+      menus.find_by(location: location, locale: default_locale)
```

Comment:
```suggestion
      menus.find_by(location: location, locale: default_locale) || menus.find_by(location: location)
```

so if there's no menu with this locale fallback the the first one?

Reply:
Now returns the first created menu for the location as a last, last resort: `find_by(location: location_name, created_at: :asc)`

Reply:
`default_scope { order(created_at: :asc) }` seems to work.

Reply:
@MatthewKennedy I also think it's work moving this line here: https://github.com/spree/spree/pull/11051/files#diff-28063c4275ea907ab47a79d37844ab42ca50fe09f376f958745615b3b8d45363R36

It seems to me this belongs to being the "default menu". WDYT?

Reply:
Hi @mad-eel 

Yeah I'll move that over and test it out, it is a busy section of code that.